lib/archethic/utils/regression/faucet_provider.ex

```diff
@@ -0,0 +1,7 @@
+defmodule ArchEthic.Utils.Regression.Benchmark.FaucetProvider do
```

Comment:
Why are you creating this behavior, I think the faucet funding should be a kind "helper" for "TPS" benchmark.